### PR TITLE
Remove unneeded conftest imports and add documentation in test_execd and test_agentd

### LINKED ISSUE #1071: Remove import conftest in integration tests
Some tests have unnecessary conftest module import. 

```
from conftest import *
```

This could cause errors in documentation generation along with scalability issues. It should be desirable to remove them.

List of tests with this import:
 - ` tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py`
 -  `tests/integration/test_active_response/test_execd/test_execd_restart.py`
 -  `tests/integration/test_agentd/test_agent_auth_enrollment.py`
 - ` tests/integration/test_agentd/test_agentd_enrollment_params.py`
 -  `tests/integration/test_agentd/test_agentd_multi_server.py`
 -  `tests/integration/test_agentd/test_agentd_parametrized_reconnections.py`
 -  `tests/integration/test_agentd/test_agentd_reconnection.py`

# UPDATE

In addition, the following tasks will be performed on the above tests:
 -  Check for compliance with PEP-8 style guide.
 -  Add missing docstrings.
 -  Add missing documentation in wazuh-qa/doc.

### PR DESCRIPTION
**Pull request by mdengra**
|Related issue|
|---|
|Closes #1071|

## Description

This PR removes the `conftest` module imports in the execd and agentd tests referenced in #1071. It also adds the missing docstrings and documentation (including indexes) in those tests.

To remove the `conftest` imports in `test_execd` it has been necessary to transfer the auxiliary functions contained in it to a new module in `wazuh_testing` called `execd.py` (according to convention it should have been called `exec.py`, but this is a reserved name).

The `conftest` imports in test_agentd have been removed by relocating the auxiliary functions from it to the `agent.py` module of `wazuh_testing`.

## Tests

- [x] Proven that tests **pass** when they have to pass.
- [x] Proven that tests **fail** when they have to fail.

- [x] Python codebase satisfies PEP-8 style style guide. `pycodestyle --max-line-length=120 --show-source --show-pep8 file.py`.
- [x] Python codebase is documented following the Google Style for Python docstrings.
- [x] The test is documented in wazuh-qa/docs.
- [x] `provision_documentation.sh` generate the docs without errors.

### CODE REVIEW

File: docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+
+## Overview
+
+These tests will check that the connection is finally made when
+there are delays between connection attempts to the server.
+
+## Objective
+
+The objective is to check how AgentD behaves when there are delays between connection attempts
+to RemoteD using `TCP` and `UDP` protocols.
```
Comment (BraulioV) [inline docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md:11]:
```suggestion
The objective is to check how wazuh-agentd behaves when there are delays between connection attempts
to wazuh-remoted using `TCP` and `UDP` protocols.
```

File: docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+| 0 | Linux/Windows | 9 | 18m 39s |
+
+## Expected behavior
+
+Success if the agent connects with the manager, failure otherwise.
+
+## Testing
+
+The tests are based on testing a different number of connection attempts and time between
+them and then, verifying that the connection between AgentD and RemoteD is finally successful.
```
Comment (BraulioV) [inline docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md:26]:
```suggestion
them and then, verify if the connection between wazuh-agentd and wazuh-remoted is finally successful. 
```

File: docs/tests/integration/test_agentd/test_agentd_reconnection.md [hunk 3]
```diff
@@ -1,3 +1,44 @@
+# Test AgentD reconnection
```
Comment (BraulioV) [inline docs/tests/integration/test_agentd/test_agentd_reconnection.md:1]:
```suggestion
# Test wazuh-agentd reconnection
```

File: docs/tests/integration/test_agentd/test_agentd_reconnection.md [hunk 4]
```diff
@@ -1,3 +1,44 @@
...
+
+## Overview
+
+These tests will check that, during enrollment, the agent re-establishes communication with the manager
```
Comment (BraulioV) [inline docs/tests/integration/test_agentd/test_agentd_reconnection.md:5]:
```suggestion
These tests will check if, during enrollment, the agent re-establishes communication with the manager
```

File: docs/tests/integration/test_agentd/test_agentd_reconnection.md [hunk 5]
```diff
@@ -1,3 +1,44 @@
...
+|:--:|:--:|:--:|:--:|
+| 0 | Linux/Windows | 10 | 17m 53s |
+
+## Expected behavior
+
+Success if the agent enrolls and starts communication with remoted, failure otherwise.
+
+## Testing
+
+The tests are based on verifying that AgentD is communicating with RemoteD (if the agent is already enrolled),
```
Comment (BraulioV) [inline docs/tests/integration/test_agentd/test_agentd_reconnection.md:25]:
```suggestion
The tests are based on verifying that wazuh-agentd is communicating with wazuh-remoted (if the agent is already enrolled),
```

File: tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py [hunk 6]
```diff
@@ -5,20 +5,18 @@
...
 from wazuh_testing.tools import WAZUH_PATH, LOG_FILE_PATH
 from wazuh_testing.tools.authd_sim import AuthdSimulator
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.file import truncate_file
 from wazuh_testing.tools.monitoring import FileMonitor
 from wazuh_testing.tools.remoted_sim import RemotedSimulator
 from wazuh_testing.tools.services import control_service
-
-from conftest import AR_LOG_FILE_PATH, wait_received_message_line, wait_start_message_line, \
+from wazuh_testing.execd import AR_LOG_FILE_PATH, wait_received_message_line, wait_start_message_line, \
```
Comment (BraulioV) [inline tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py:19]:
```suggestion
import wazuh_testing.execd as exec
# ....
exec.AR_LOG_FILE_PATH
```

File: tests/integration/test_active_response/test_execd/test_execd_restart.py [hunk 7]
```diff
@@ -20,9 +15,8 @@
 from wazuh_testing.tools.monitoring import FileMonitor
 from wazuh_testing.tools.authd_sim import AuthdSimulator
 from wazuh_testing.tools.remoted_sim import RemotedSimulator
-from conftest import AR_LOG_FILE_PATH, set_ar_conf_mode, set_debug_mode, \
-    wait_received_message_line, wait_start_message_line, \
-    wait_ended_message_line, test_version, start_log_monitoring
+from wazuh_testing.execd import AR_LOG_FILE_PATH, wait_received_message_line, \
+    wait_start_message_line, wait_ended_message_line, start_log_monitoring
```
Comment (BraulioV) [inline tests/integration/test_active_response/test_execd/test_execd_restart.py:19]: Same as before. This improves the readability of the script.

File: tests/integration/test_agentd/test_agent_auth_enrollment.py [hunk 8]
```diff
@@ -2,15 +2,18 @@
...
 import subprocess
 import yaml
 
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.authd_sim import AuthdSimulator
 from wazuh_testing.tools.monitoring import QueueMonitor
-from conftest import *
+from wazuh_testing.agent import AGENT_AUTH_BINARY_PATH, AUTHDPASS_PATH, DEFAULT_VALUES, \
+    SERVER_CERT_PATH, SERVER_KEY_PATH, AgentAuthParser, build_expected_request, \
+    check_client_keys_file, configure_enrollment, parse_configuration_string
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agent_auth_enrollment.py:16]:
```suggestion
import wazuh_testing.agent as ag
...
ag.AGENT_AUTH_BINARY_PATH
```

File: tests/integration/test_agentd/test_agent_auth_enrollment.py [hunk 9]
```diff
@@ -21,10 +24,14 @@
...
-    ----------
-    yaml structure
+    """
+    Load a yaml file from a path.
+
+    Args:
+        path: File location.
+
+    Returns:
+        yaml structure.
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agent_auth_enrollment.py:34]:
```suggestion
    """Load a YAML file from a path.

    Args:
        path: File location.

    Returns:
        dict: dictionary with the info from the YAML.
```

File: tests/integration/test_agentd/test_agentd_enrollment_params.py [hunk 10]
```diff
@@ -3,18 +3,20 @@
...
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.configuration import set_section_wazuh_conf, write_wazuh_conf
 from wazuh_testing.tools.monitoring import QueueMonitor, FileMonitor
 from wazuh_testing.tools.remoted_sim import RemotedSimulator
 from wazuh_testing.tools.services import control_service
-
-from conftest import *
+from wazuh_testing.agent import CLIENT_KEYS_PATH, DEFAULT_VALUES, SERVER_CERT_PATH, SERVER_KEY_PATH, \
+    AgentAuthParser, build_expected_request, check_client_keys_file, clean_client_keys_file, \
+    clean_password_file, parse_configuration_string, configure_enrollment
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_enrollment_params.py:19]: Same as before

File: tests/integration/test_agentd/test_agentd_enrollment_params.py [hunk 11]
```diff
@@ -27,10 +29,14 @@
...
-    ----------
-    yaml structure
+    """
+    Load a yaml file from a path.
+
+    Args:
+        path: File location.
+
+    Returns:
+        yaml structure.
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_enrollment_params.py:39]: Same comment than before.

File: tests/integration/test_agentd/test_agentd_enrollment_params.py [hunk 12]
```diff
@@ -86,6 +95,17 @@ def clean_log_file():
...
+    """
+    It applies custom settings on ossec.conf file from values located under
+    "configuration" section of tests found in a YAML file. For this purpose,
+    it stops the wazuh-agentd service, applies the settings and starts it again.
+
+    Args:
+        configuration: New parameters to be applied.
+
+    Raises:
+        Exception: If wazuh-agentd daemon cannot be started again.
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_enrollment_params.py:107]:
```suggestion
        ValueError: If wazuh-agentd daemon cannot be started again.
```

File: tests/integration/test_agentd/test_agentd_enrollment_params.py [hunk 13]
```diff
@@ -86,6 +95,17 @@ def clean_log_file():
 
 
 def override_wazuh_conf(configuration):
+    """
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_enrollment_params.py:98]: Add a first line that explains what it does in a single line. If you need more detail, split it and add a new paragraph.

File: tests/integration/test_agentd/test_agentd_enrollment_params.py [hunk 14]
```diff
@@ -110,11 +130,20 @@ def override_wazuh_conf(configuration):
     try:
         # Start Wazuh
         control_service('start', daemon='wazuh-agentd')
-    except:
+    except ValueError:
         raise Exception
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_enrollment_params.py:134]: Remove the `try/except` block. Just run `control_service` and let run the exception if something happens.

File: tests/integration/test_agentd/test_agentd_enrollment_params.py [hunk 15]
```diff
@@ -86,6 +95,17 @@ def clean_log_file():
 
 
 def override_wazuh_conf(configuration):
+    """
+    It applies custom settings on ossec.conf file from values located under
+    "configuration" section of tests found in a YAML file. For this purpose,
+    it stops the wazuh-agentd service, applies the settings and starts it again.
+
+    Args:
+        configuration: New parameters to be applied.
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_enrollment_params.py:104]: The type is missing here.

File: tests/integration/test_agentd/test_agentd_parametrized_reconnections.py [hunk 16]
```diff
@@ -47,23 +49,26 @@
...
+    Set debug2 for agentd in local internal options file.
+    """
     if platform.system() == 'win32' or platform.system() == 'Windows':
         local_int_conf_path = os.path.join(WAZUH_PATH, 'local_internal_options.conf')
         debug_line = 'windows.debug=2\n'
     else:
         local_int_conf_path = os.path.join(WAZUH_PATH, 'etc', 'local_internal_options.conf')
         debug_line = 'agent.debug=2\n'
-    with  open(local_int_conf_path, 'r') as local_file_read:
+    with open(local_int_conf_path, 'r') as local_file_read:
```
Comment (BraulioV) [inline tests/integration/test_agentd/test_agentd_parametrized_reconnections.py:66]:
```suggestion
    with open(local_int_conf_path) as local_file_read:
```

**Review by BraulioV — CHANGES_REQUESTED** [review]

File: docs/tests/integration/test_active_response/index.md [hunk 17]
```diff
@@ -1 +1,16 @@
-# Overview
\ No newline at end of file
+# Active response tests
+
+# Overview
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_active_response/index.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_active_response/test_execd/index.md [hunk 18]
```diff
@@ -1 +1,25 @@
-# Overview
\ No newline at end of file
+# Test execd
+
+## Overview 
+
+Active responses execute a script in response to the triggering of specific alerts
+based on the alert level or rule group.
+The `wazuh-execd` program runs Active Responses by initiating the configured scripts.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_active_response/test_execd/index.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_active_response/test_execd/index.md [hunk 19]
```diff
@@ -1 +1,25 @@
...
+
+These tests check if `wazuh-execd` executes the active responses correctly.
+
+## Objective
+
+Check that the Active Response scripts are executed correctly by `execd`.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_active_response/test_execd/index.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_active_response/index.md [hunk 20]
```diff
@@ -1 +1,16 @@
...
+to an agent from the threat source when certain criteria are met.
+
+## Tiers
+### Tier 0
+#### Test analysisd
+
+
+#### Test execd
+
+These tests will check that the active responses, which are executed by the `wazuh-execd` program via scripts,
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_active_response/index.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: tests/integration/test_agentd/test_agentd_parametrized_reconnections.py (hunk 16 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_parametrized_reconnections.py]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agentd_enrollment_params.py (hunk 15 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_enrollment_params.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agentd_enrollment_params.py (hunk 13 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_enrollment_params.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agentd_enrollment_params.py (hunk 12 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_enrollment_params.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py (hunk 6 above)
Comment (mdengra) [reply tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_active_response/test_execd/test_execd_restart.py (hunk 7 above)
Comment (mdengra) [reply tests/integration/test_active_response/test_execd/test_execd_restart.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agent_auth_enrollment.py (hunk 8 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agent_auth_enrollment.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agent_auth_enrollment.py (hunk 9 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agent_auth_enrollment.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agentd_enrollment_params.py (hunk 14 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_enrollment_params.py]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/f3a48cc26993288a6948e8e952e9cd808228f5c5

File: tests/integration/test_agentd/test_agentd_enrollment_params.py (hunk 11 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_enrollment_params.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: tests/integration/test_agentd/test_agentd_enrollment_params.py (hunk 10 above)
Comment (mdengra) [reply tests/integration/test_agentd/test_agentd_enrollment_params.py]: https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb

File: docs/tests/integration/test_active_response/test_execd/test_execd_firewall_drop.md [hunk 21]
```diff
@@ -1,11 +1,12 @@
 # Test execd firewall drop
-This test check that Active Response script called 'firewall-drop' is executed correctly when configured.
+
+This test check that Active Response script called `firewall-drop` is executed correctly when configured.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_active_response/test_execd/test_execd_firewall_drop.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_active_response/test_execd/test_execd_restart.md [hunk 22]
```diff
@@ -1,11 +1,12 @@
 # Test execd restart
-This test check that Active Response script called 'restart-wazuh' is executed correctly when configured.
+
+This test check that Active Response script called `restart-wazuh` is executed correctly when configured.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_active_response/test_execd/test_execd_restart.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/index.md [hunk 23]
```diff
@@ -1 +1,37 @@
-# Overview
+# Test agentd
+
+## Overview 
+
+The `wazuh-agentd`  program is the client-side daemon that communicates with the server.
+The `agent-auth` program is the client application used along with `wazuh-authd`
+to automatically add agents to a Wazuh manager.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/index.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agent_auth_enrollment.md [hunk 24]
```diff
@@ -1,4 +1,34 @@
-## Code documentation
+# Test agent-auth enrollment
+
+## Overview
+
+These tests will verify different situations that may occur at agent-auth program during enrollment.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agent_auth_enrollment.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agent_auth_enrollment.md [hunk 25]
```diff
@@ -1,4 +1,34 @@
...
+
+## Objective
+
+The objective is to check that the parameters sent by the agent-auth program to the AuthD daemon
+are consistent with its responses.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agent_auth_enrollment.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_enrollment_params.md [hunk 26]
```diff
@@ -1,3 +1,35 @@
+# Test AgentD enrollment params
+
+## Overview
+
+These tests will verify different situations that may occur at AgentD during enrollment.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_enrollment_params.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_enrollment_params.md [hunk 27]
```diff
@@ -1,3 +1,35 @@
...
+
+## Objective
+
+The objective is to check the enrollment of the agent using certain settings
+in the `ossec.conf` file produces the expected responses from the server.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_enrollment_params.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_multi_server.md [hunk 28]
```diff
@@ -1,3 +1,56 @@
+# Test AgentD multi-server
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_multi_server.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_enrollment_params.md [hunk 29]
```diff
@@ -1,3 +1,35 @@
+# Test AgentD enrollment params
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_enrollment_params.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_multi_server.md [hunk 30]
```diff
@@ -1,3 +1,56 @@
...
+| 0 | Linux/Windows | 6 | 11m 27s |
+
+## Expected behavior
+
+Success if the agent passes the stages of each test, failure otherwise.
+
+## Testing
+
+The tests are based on simulating a multi-server environment with three
+RemoteD simulated connections that, in each test, are in a different state.
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_multi_server.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md [hunk 31]
```diff
@@ -1,3 +1,37 @@
+# Test AgentD parametrized reconnection
```
[the remark this answers is not in the document]
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md (hunk 1 above)
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md (hunk 2 above)
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_reconnection.md (hunk 3 above)
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_reconnection.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_reconnection.md (hunk 4 above)
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_reconnection.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

File: docs/tests/integration/test_agentd/test_agentd_reconnection.md (hunk 5 above)
Comment (mdengra) [reply docs/tests/integration/test_agentd/test_agentd_reconnection.md]: Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca

**Review by BraulioV — APPROVED** [review]

**Pull request merged**